.github/pull-request.md

```diff
@@ -0,0 +1,5 @@
+Closes [#link issue]
```

Comment:
I'd put this at the bottom, so the change summary is the first thing you see when looking at a pull request.